### PR TITLE
docs: fix the underline and verbosity of provider

### PR DESCRIPTION
**Pull request by Tiboris**
fix: make static provider more verbose
 

    docs: fix title underline too short

### CODE REVIEW

**Review by netoarmando — APPROVED** [review]

File: src/mrack/providers/static.py [hunk 1]
```diff
@@ -82,4 +87,5 @@ def prov_result_to_host_data(self, prov_result):
 
     async def delete_host(self, host_id):
         """Fake delete - pass but don't do anything."""
+        logger.info(f"{self.dsp_name}: Removing static host {host_id}")
```
Comment (pvoborni) [inline src/mrack/providers/static.py:90]: Static host cannot be removed by mrack. Thus the log information needs to say something like: "Host {host_id} is static - not managed by Mrack, ignoring removal."

Comment (Tiboris) [reply src/mrack/providers/static.py:90]: Will edit

**Review by pvoborni — APPROVED** [review]
ACKing  (provided that it works :) )

**Comment by f-trivino** [comment]
it works also for me. I'm merging it.

**Pull request merged**